### ISSUE TITLE
jsp: help link broken

### ISSUE THREAD

**Issue by srl295** [unicode-org/unicodetools#132]
per @PeterConstable 
> the Help link, which goes to https://cldr.unicode.org/unicode-utilities/, is broken.

Also note #56

**Comment by markusicu** [comment]
It might be one of the URLs that Annemarie has not yet restored. There should be a page with the contents of what used to be under that URL.

**Comment by macchiati** [comment]
Yes, sometimes the URLs had to be changed. Steven, I just do a search on
the site, eg

https://cldr.unicode.org/_/search?query=unicode%20utilities&scope=site&showTabs=false

That leads right to https://cldr.unicode.org/development/unicode-utilities

(A lot of those pages need work, but we can at least have the links work.)

Mark

**Comment by AEApple** [comment]
It should be fixed now. Please  check and let me know if anything is still broken.

**Comment by PeterConstable** [comment]
Works now. Thanks.

**Comment by PeterConstable** [comment]
Oops, spoke too soon. The help link works from this page,

https://util.unicode.org/UnicodeJsps/

but not from pages for the individual utilities:

https://util.unicode.org/UnicodeJsps/character.jsp
https://util.unicode.org/UnicodeJsps/properties.jsp
https://util.unicode.org/UnicodeJsps/confusables.jsp
https://util.unicode.org/UnicodeJsps/list-unicodeset.jsp
https://util.unicode.org/UnicodeJsps/unicodeset.jsp
https://util.unicode.org/UnicodeJsps/regex.jsp
https://util.unicode.org/UnicodeJsps/bnf.jsp
https://util.unicode.org/UnicodeJsps/breaks.jsp
https://util.unicode.org/UnicodeJsps/transform.jsp
https://util.unicode.org/UnicodeJsps/bidi.jsp
https://util.unicode.org/UnicodeJsps/bidic.jsp
https://util.unicode.org/UnicodeJsps/idna.jsp
https://util.unicode.org/UnicodeJsps/languageid.jsp


Peter

From: Peter Constable ***@***.***>
Sent: Friday, October 1, 2021 5:49 PM
To: unicode-org/unicodetools ***@***.***>; unicode-org/unicodetools ***@***.***>
Cc: Peter Constable ***@***.***>; Mention ***@***.***>
Subject: RE: [unicode-org/unicodetools] jsp: help link broken (#132)

Works now. Thanks.

**Comment by AEApple** [comment]
Fixed -> let me know if any still don't work. Thanks!
